### PR TITLE
READY TO MERGE

### PR DESCRIPTION
**Pull request by tleskin**

### CODE REVIEW

File: students/tom_leskin/20150814-tom_leskin-m4.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+* I delivered 100% of the requested feedback
+* 96% of those submissions were approved and delivered
+* 84% of my submissions were meaningful to the recipient
+
+### Playing a Part
+
+* On Friday, July 10, I delivered a lightning talk "Prototypal vs Classical Inheritance"
+* On Friday, July 10, I met with some 1507 people to answer some questions about Atom
+* On Saturday, July 18, I assisted Rachel as a TA at an Intro to Ruby class with Girl Develop It at Twitter Boulder
```
Comment (kristinabrown) [inline students/tom_leskin/20150814-tom_leskin-m4.md]: Tom! You are great. Your portfolio is looking solid. You're a quiet, reserved little unicorn and your magical presence has made Turing a better place :horse:

**Comment by loriculberson** [comment]
Mr. Calm, Cool and Collected. I appreciate how you are always willing to lend a helping hand. It's always great to pair with you. You do a good job explaining..... this; and other JavaScript related topics.

**Comment by MichelleGolle** [comment]
I always enjoy working with you and after pairing on Game Time I think you've definitely met your goal of improving your javascript skills. It's also really cool that you've made time to mentor outside of school.

**Comment by patrickmedaugh** [comment]
Tom, I love how Turing has brought you out of your shell and how you've opened up since module 1. You're a great team member to work with and you really keep up to date in the tech world. #freestuffandpajamajeans4lyf

**Comment by tleskin** [comment]
Ready to merge, aside from any final eval feedback from Jeff.

**Comment by boomkenster** [comment]
TOMMY! 

Tom has excelled every module. In module one I remember Tom being oblivious at times but now he can do any task and believes in himself. He has incredible work ethic, amazingly diligent in his process and research and resilient in his efforts.

**Comment by tleskin** [comment]
Ready!

**Pull request merged**